logzio/sender.py

```diff
@@ -34,14 +34,18 @@ def __init__(self, token, url="https://listener.logz.io:8071", logs_drain_timeou
 
         # Create a queue to hold logs
         self.queue = queue.Queue()
+        self._initialize_sending_thread()
```

Comment:
Maybe let's leave the initialization in the c'tor, and just call "start()" in another method? seems more intuitive

Reply:
Yea, good suggestion, thanks!

Reply:
Pushed that and didn't check the build...

I don't think that works, so I reverted. 

I agree its more intuitive. Encapsulating this in a method is not counter intuitive (makes the constructor more concise) and I think we need a new thread instance if it's not alive. See message in c6c8171 for some more comments